### ISSUE TITLE
Syntax Highlighting gets weird when I use optional chaining

### ISSUE THREAD

**Issue by maisans-maid** [atom/atom#21836]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description
As the title says, the syntax highlighting doesn't get it right when i use feaatures from es2020 like optional chaining
![image](https://user-images.githubusercontent.com/56829176/103385078-f3c32580-4b33-11eb-85c7-b26148cd59e7.png)

It works again as expected when i remove it
![image](https://user-images.githubusercontent.com/56829176/103385149-2ff68600-4b34-11eb-9643-7260943dcee5.png)

### Steps to Reproduce

1. write code
2. attempt to use optional chaining
3. expect the wrong syntax highlighting

**Expected behavior:**

Correct syntax highlighting

**Actual behavior:**

Wrong syntax highlighting

**Reproduces how often:**
100%

### Versions

atom version: 1.53.0 x64
OS: Windows 10 Version 10.0.18363.1256

**Comment by MoonlightCapital** [comment]
I can reproduce.

Atom 1.53.0
Electron 6.1.12
Chrome 76.0.3809.146
Node v12.4.0

My code uses node v14.15.3

**Comment by ThatXliner** [comment]
This should be in the language-JavaScript package, no?

**Comment by maisans-maid** [comment]
> This should be in the language-JavaScript package, no?

yes, i found a pr there that seems to fix this issue at https://github.com/atom/language-javascript/pull/686
